<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="4">
      <widget class="QComboBox" name="cmbBoxUnitsWeight">
       <item>
        <property name="text">
         <string>kg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>st</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="lblGender">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QSpinBox" name="spnBoxHeight"/>
     </item>
     <item row="1" column="2" colspan="3">
      <widget class="QComboBox" name="cmbBoxGender">
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="lblHeight">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QComboBox" name="cmbBoxActivityLevel">
       <item>
        <property name="text">
         <string>Sedendary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lightly active</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moderately active</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very active</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extra active</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lblAge">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QSpinBox" name="spnBoxAge"/>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QSpinBox" name="spnBoxWeight"/>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="cmbBoxUnitsHeight">
       <item>
        <property name="text">
         <string>cm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>in</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="lblActivityLevel">
       <property name="text">
        <string>Activity level:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="lblWeight">
       <property name="text">
        <string>Weight:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnCalculate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chkBoxAdvancedSettings">
       <property name="text">
        <string>Advanced Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpBoxAdvancedSettings">
       <property name="title">
        <string>Results unit:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radBtnCalories">
            <property name="text">
             <string>Calories</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radBtnKilojoules">
            <property name="text">
             <string>Kilojoules</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
